plugins/default.nix

```diff
@@ -23,6 +23,7 @@
 
     ./languages/ledger.nix
     ./languages/nix.nix
+    ./languages/plantuml_syntax.nix
```

Comment:
Can you rename the file to `plantuml-syntax.nix` instead? That more closely matches the rest of the project

Reply:
Renamed!